.storybook/preview.tsx

```diff
@@ -0,0 +1,23 @@
+import type { Preview } from '@storybook/react'
+import React from 'react'
```

Comment:
remove unused import plz

Reply:
https://stackoverflow.com/a/70122171

Reply:
https://dev.to/titungdup/you-no-longer-need-to-import-react-from-react-3pbj

in this pr i changed it and it's works for me without this import 
https://github.com/hasadna/open-bus-map-search/pull/166

Reply:
doesn't work in the storybook thingy. maybe the TS version isn't up to date. it can be improved in future PR